packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -81,7 +81,7 @@ struct CreateDataTypeRequest {
 async fn create_data_type<P: StorePool + Send>(
```

Comment:
(comment for line 73)
we need `body = PersistedOntologyIdentifier` and the same changes for the rest of the endpoints :)

Reply:
(and for line 34)
The new types need to be added to the components list. This will make it so you have to run `cargo make generate-openapi-client` to update the API.

Reply:
Applied in 17d59680df2907e253a77b00387dd1d358e5cfb0 and ab1ab9926b27e0804bef3cd4beab13178b1daea2